pycalphad/io/tdb.py

```diff
@@ -238,7 +238,8 @@ def _tdb_grammar(): #pylint: disable=R0914
     # PHASE
     cmd_phase = TCCommand('PHASE') + symbol_name + \
         Suppress(White()) + CharsNotIn(' !', min=1) + Suppress(White()) + \
-        Suppress(int_number) + Group(OneOrMore(float_number)) + LineEnd()
+        Suppress(int_number) + Group(OneOrMore(float_number)) + \
```

Comment:
I'm okay to leave this in but it should be documented separately that this commit is for MatCalc compatibility.

Reply:
I hadn’t intended to sneak this in here, that commit got swept in by accident. 

I can leave it in or take it out. However, I do want to note that this is actually a documented feature for the PHASE keyword in the Thermo-Calc documentation. Anything after the sublattice numbers is an “auxiliary text string”

Reply:
Leaving it is fine.